salt/auth/__init__.py

```diff
@@ -330,6 +330,7 @@ def authenticate_key(self, load, key):
                 return auth_user.sudo_name()
             elif load['user'] == self.opts.get('user', 'root') or load['user'] == 'root':
                 if auth_key != key[self.opts.get('user', 'root')]:
+                    log.warning('Master runs as "{}", but user in payload is "{}"'.format(self.opts.get('user', 'root'), load['user']))
```

Comment:
```suggestion
                    log.warning('Master runs as %r, but user in payload is %r', self.opts.get('user', 'root'), load['user'])
```

Salt prefers log formatting for logs. I'm surprised that the linter didn't complain about this one :suspect: Usually it does.